books/free-programming-books.md

```diff
@@ -2575,7 +2575,9 @@ Kerridge (PDF) (email address *requested*, not required)
 #### Pandas
 
 * [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.18.1)
-* [pandas: powerful Python data analysis toolkit](https://pandas.pydata.org/docs/pandas.pdf) - Wes McKinney and the Pandas Development Team (:construction: *in process*) (PDF)
+* [pandas: powerful Python data analysis toolkit](https://pandas.pydata.org/docs) - Wes McKinney and the Pandas Development Team
+  * [HTML zipped format](https://pandas.pydata.org/docs/pandas.zip) (HTML)
```

Comment:
Is enough this or I should remove HTML/PDF?

Reply:
usually best to have one link go to a page where multiple formats are available. Sometimes that doesn't exist, or it has multiple books.

Reply:
Done!